platform/security/rest/security-rest-cxfwrapper/src/main/java/org/codice/ddf/cxf/client/impl/SecureCxfClientFactoryImpl.java

```diff
@@ -648,11 +642,6 @@ public final T getClientForSystemSubject(Subject subject) {
                   && oauthSecurity != null) {
                 oauthSecurity.setSystemTokenOnClient(
                     WebClient.client(newClient), clientId, clientSecret, discoveryUrl);
-              } else if (!basicAuth
```

Comment:
Does this mean we also need to remove the SAML federation option from the sources? Cause the only options look like they are OAuth or Basic.

Reply:
The only SAML thing on sources should now be SAML ECP